tensorflow/c/tf_tensor.cc

```diff
@@ -182,7 +182,17 @@ void TF_TensorBitcastFrom(const TF_Tensor* from, TF_DataType type,
 
 namespace tensorflow {
 
-void TensorInterface::Release() { delete this; }
+void TensorInterface::Release() {
+  if (Type() == TF_STRING) {
```

Comment:
The internal build errors out with 

third_party/tensorflow/c/tf_tensor.cc:186:14: error: comparison of different enumeration types ('tensorflow::DataType' and 'TF_DataType') [-Werror,-Wenum-compare]
  if (Type() == TF_STRING) {
      ~~~~~~ ^  ~~~~~~~~~
1 error generated.

I think the RHS needs to be DT_STRING instead of TF_STRING

Reply:
fixed